docs/wiki/deployment/kafka-producer.md

```diff
@@ -0,0 +1,23 @@
+Logging support utilizing a Kafka producer.
```

Comment:
Do you mind merging this into a subsection of the logger page? I know AWS is a standalone page, but I intend on combining that too.